docs/cristin.ttl

```diff
@@ -232,70 +289,93 @@
 
 :institusjonsnr
     a            rdf:Property ;
-    rdfs:comment "The Cristin institution identifier of the respective institution." ;
+    rdfs:comment """
+    The **current** Cristin institution identifier institution that the contributor of the publication was affiliated with.
```

Comment:
Does this mean the user's current institution, or the current identifier for the historical institution, e.g. the inheriting institution?

Reply:
Similarly, the other similar descriptions